### PR TITLE
fix: Doc about installing ImagePuller

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
fix: Doc about installing ImagePuller

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-7865

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc [hunk 1]
```diff
@@ -9,15 +9,19 @@
 
 You can install the {image-puller-name} on OpenShift by using OpenShift `oc` management tool.
 
+[IMPORTANT]
+====
+If the ImagePuller is installed with the `oc` CLI, it cannot be configured via the `CheCluster` Custom Resource.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc:14]:
```suggestion
If the ImagePuller was installed with the `oc` CLI, you can not configure it via the `CheCluster` Custom Resource.
```

**Review by deerskindoll — COMMENTED** [review]
small tweak

**Review by deerskindoll — APPROVED** [review]
lgtm

**Comment by tolusha** [comment]
Some tests fails, can I merge anyway ?

**Comment by deerskindoll** [comment]
> Some tests fails, can I merge anyway ?

please cherry-pick this PR: https://github.com/eclipse-che/che-docs/pull/2836

it solves the issue with the failing htmltest

**Pull request merged**